### ISSUE TITLE
Using URL-PlayWriter/PlayWright Browser Toolkit in Jupyter Notebook 

### ISSUE THREAD

**Issue by kanik9** [langchain-ai/langchain#8342]
### System Info

While using both URL-Playwriter or PlayWright Browser Toolkit I am getting this error. 
The solution I used:
1. I used asyncio.create_task
2. I used nest_asyncio.apply
but the error is the same. Please help if someone is also getting the same error and found some solution 

![image](https://github.com/langchain-ai/langchain/assets/32274469/a2c30690-46eb-454c-a954-1ef9d23ef150)

### Information

- [ ] The official example notebooks/scripts
- [ ] My own modified scripts

### Related Components

- [ ] LLMs/Chat Models
- [ ] Embedding Models
- [ ] Prompts / Prompt Templates / Prompt Selectors
- [ ] Output Parsers
- [X] Document Loaders
- [ ] Vector Stores / Retrievers
- [ ] Memory
- [ ] Agents / Agent Executors
- [X] Tools / Toolkits
- [ ] Chains
- [ ] Callbacks/Tracing
- [X] Async

### Reproduction

![image](https://github.com/langchain-ai/langchain/assets/32274469/e2bb198a-558e-43f9-825d-31af6f36bf83)
![image](https://github.com/langchain-ai/langchain/assets/32274469/7f54eaf1-f1ad-40da-ac8f-8f7f4d6e85c6)

### Expected behavior

This code needs to return the documents of the html content, but I am getting error.

**Comment by kanik9** [comment]
I perform each and every step that is mentioned in the docs but I am getting this 

I think that we are running multiple event loops
1. the first loop is running by the Jupyter-notebook 
2. second one was created by the long-chain developers 
3. third one which I need to create to run the async playwriter thing

**Comment by kanik9** [comment]
Langchain version which I use is: langchain==0.0.242 and langchain==0.0.244
The solution which I used till now are:
1. I already try asyncio.run()
2. I used nest_asyncio.apply()
3. I also tried task creation using asyncio
4. In short I tried all the solutions which are available on the Web to run asyncio code on Jupyter Notebook but did not succeed 

Today I tried to run this with FastAPI same error occur


I think the problem is inside the PlaywrightURLLoader
